### DISCUSSION TITLE
Add ranging for SX1280

### DISCUSSION THREAD

**Discussion by JBKingdon** [ExpressLRS/ExpressLRS#1337] [Ideas]
The SX1280 has a neat capability to do time of flight measurement to estimate distance between the tx and rx. This could be really useful for locating a fail-safed quad in tall vegetation for example.

**Comment by maybenikhil** [comment]
https://semtech.my.salesforce.com/sfc/p/#E0000000JelG/a/44000000MDiH/OF02Lve2RzM6pUw9gNgSJXbDNaQJ_NtQ555rLzY3UvY

https://semtech.my.salesforce.com/sfc/p/#E0000000JelG/a/44000000MDcY/ZsmAVCVenZkc0lUrr3RuxWSfdFxY2Tj_msk4N9DAhBo

**Comment by brandonrc** [comment]
@JBKingdon I know this is an old issue but I think one of the down sides is the reduced range when you go into that mode. 

Another down side is if a company uses sx1281 chips that do not support this functionality. 

There was an interesting idea about locating based on time it takes to get a telemetry packet. Not sure how accurate that would be. https://github.com/ExpressLRS/ExpressLRS/issues/892

**Reply by gauravduggal** [reply]
This would be an excellent feature. Is anyone interested in this? I would be willing to develop this with some collaboration if there is any interest.
